src/pages/messages/components/NviCandidatePage.tsx

```diff
@@ -37,6 +47,15 @@ export const NviCandidatePage = () => {
 
   const periodStatus = nviCandidate?.periodStatus.status;
 
+  const nextCandidateQuery = useQuery({
+    queryKey: ['nextCandidate', 1, offsetNextCandidate, nviListQuery],
+    queryFn: offsetNextCandidate ? () => fetchNviCandidates(1, offsetNextCandidate, `${nviListQuery}`) : undefined,
+    meta: { errorMessage: false },
+    retry: false,
+  });
+
+  const nextCandidateIdentifier = nextCandidateQuery.data?.hits?.[0]?.identifier;
```

Comment:
Om vi har `data` vil vi også ha `hits` (men den kan være tomt array)
```suggestion
  const nextCandidateIdentifier = nextCandidateQuery.data?.hits[0]?.identifier;
```